src/main/java/servlets/LoginServlet.java

```diff
@@ -53,17 +47,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   /** Build the OAuth consent page redirect URL. */
   private String getOAuthRedirectURL() {
-    String redirectUri = REDIRECT_URI + getRedirectUri();
-    
-    return String.format("%s?%s&%s&%s&%s", OAUTH_LOGIN_URI, CLIENT_ID,
-      RESPONSE_TYPE, redirectUri, SCOPE); 
+    return String.format("%s?%s&%s&%s&%s", OAuthConstants.OAUTH_LOGIN_URI, 
+      OAuthConstants.CLIENT_ID, OAuthConstants.RESPONSE_TYPE, getRedirectUri(), 
+      OAuthConstants.SCOPE); 
```

Comment:
can you put an example in a comment of what this string will look like?

Reply:
+1 for an example of what this string will look like.